src/app/loginComponents/requests/requests.component.ts

```diff
@@ -118,4 +122,28 @@ export class RequestsComponent extends Base implements OnInit {
   rerequestReview(membership: OrganizationUser) {
     this.requestsService.requestRereview(membership.organization.id);
   }
+
+  removeOrganizationDialog(organizationName: string, organizationStatus: string, organizationID: number) {
+    const confirmationDialogData: ConfirmationDialogData = {
+      title: 'Delete Organization',
+      message: `Are you sure you wish to delete this ${organizationStatus} organization?
```

Comment:
In your screenshot the status is all caps, would be nicer if it were lowercase.

Reply:
Done. Changed the status to lowercase in the message.